Makefile

```diff
@@ -17,7 +17,7 @@ ifeq ($(OS),Darwin)
 endif
 
 ifeq ($(OS),Linux)
-	LD_LIBRARY_PATH += $(CURDIR)/operator/risc_zero/lib
+	LD_LIBRARY_PATH+=$(CURDIR)/operator/risc_zero_old/lib:$(CURDIR)/operator/risc_zero/lib:$(LD_LIBRARY_PATH)
```

Comment:
This assignment should be `:=` if it appends the original value. Otherwise, it creates a recursive variable that references itself.